### ISSUE TITLE
Error Exporting Android AAB bundle 

### ISSUE THREAD

**Issue by MrZak-dev** [defold/defold#4974]
**Describe the bug**
Build error when exporting a game to android AAB format

```
Failed building Android Application Bundle: Picked up JAVA_TOOL_OPTIONS: 
Picked up _JAVA_OPTIONS: 
[BT:0.14.0] Error: Module 'base' is missing mandatory file 'manifest/AndroidManifest.xml'.
com.android.tools.build.bundletool.model.exceptions.BundleFileTypesException$MandatoryModuleFileMissingException: Module 'base' is missing mandatory file 'manifest/AndroidManifest.xml'.
	at com.android.tools.build.bundletool.validation.MandatoryFilesPresenceValidator.checkModuleHasAndroidManifest(MandatoryFilesPresenceValidator.java:80)
	at com.android.tools.build.bundletool.validation.MandatoryFilesPresenceValidator.validateModuleZipFile(MandatoryFilesPresenceValidator.java:41)
	at com.android.tools.build.bundletool.validation.ValidatorRunner.lambda$validateModuleZipFile$2(ValidatorRunner.java:53)
	at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:406)
	at com.android.tools.build.bundletool.validation.ValidatorRunner.validateModuleZipFile(ValidatorRunner.java:53)
	at com.android.tools.build.bundletool.validation.BundleModulesValidator.validate(BundleModulesValidator.java:81)
	at com.android.tools.build.bundletool.commands.BuildBundleCommand.execute(BuildBundleCommand.java:208)
	at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:71)
	at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:47)
```

**To Reproduce**
1. Create New Project 
2. Export To AAB

**Defold version :**
  - Version 1.2.170

**Platforms (REQUIRED):**
 - Platforms:  Android,
 - OS: Windows 10

**Screenshots :**

![image](https://user-images.githubusercontent.com/27586088/86001759-0a1ee600-ba18-11ea-835f-9a4b6194fd79.png)

**Comment by AGulev** [comment]
Could you please share a screenshot of Android section from game.project  file

**Comment by MrZak-dev** [comment]
![image](https://user-images.githubusercontent.com/27586088/86005752-b1524c00-ba1d-11ea-8dcd-bf6edc92a3ca.png)

**Comment by britzl** [comment]
Possibly related: https://github.com/google/bundletool/issues/80

It could be that the zip implementation differs between OS versions. Need to look into this step: https://github.com/defold/defold/blob/dev/com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAAB.java#L372-L379

**Comment by britzl** [comment]
@MrZak-dev this has been fixed on the dev branch and is available as an alpha: http://d.defold.com/alpha/

Closed by #4777
